### ISSUE TITLE
[jit] Can't script .type() 

### ISSUE THREAD

**Issue by hx89** [pytorch/pytorch#25433]
## To Reproduce

```
import torch
x = torch.rand(3, 4)
def foo(x):
    return x.type(torch.int8)
scr = torch.jit.script(foo)
print(scr(x))
```
Got error:
```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-10-1c48db585630> in <module>
      6     return x.type(torch.int8)
      7 
----> 8 scr = torch.jit.script(foo)
      9 
     10 print(scr(x))

/mnt/xarfuse/uid-136047/39b8cc66-ns-4026531840/torch/jit/__init__.py in script(obj, optimize, _frames_up, _rcb)
   1180         if _rcb is None:
   1181             _rcb = _gen_rcb(obj, _frames_up)
-> 1182         fn = torch._C._jit_script_compile(qualified_name, ast, _rcb, get_default_args(obj))
   1183         # Forward docstrings
   1184         fn.__doc__ = obj.__doc__

RuntimeError: 
Unknown builtin op: aten::type.
Here are some suggestions: 
	aten::topk
	aten::eye
	aten::take

The original call is:
at <ipython-input-10-1c48db585630>:6:11
def foo(x):
    return x.type(torch.int8)
           ~~~~~~ <--- HERE 
```

cc @suo

**Comment by eellison** [comment]
Thanks for the report! We should fix this but as a workaround until then you can use `x.to(torch.int8)`

**Comment by nikithamalgifb** [comment]
This is fixed now by #51904. Feel free to re-open if the issue persists
